CannibalisticZombies/Assets/Scripts/Character/Player/BulletCollision.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace CannibalisticZombies {
+    public class BulletCollision : MonoBehaviour
+    {
+
+        /// -////////////////////////////////////////////////////////////////////
+        /// author: Ashley Roman
+        /// combat system
+        /// 
+
+        [SerializeField] protected float damage = 5f;
+
+        private void OnTriggerEnter(Collider collision)
+        {
+            if (collision.gameObject.CompareTag("Enemy"))
```

Comment:
Instead of comparing tag, you can actually use `enemy` that you got from the code below and do a null check!
`EnemyBase enemy = collision.gameObject.GetComponent<EnemyBase>();`

I suggest because I think it is cleaner and doesn't require for all enemies to be tagged (Which can be easy to miss).
If you do go this route, make sure to delete the enemy tag